lib/matplotlib/tight_layout.py

```diff
@@ -1,3 +1,16 @@
 from matplotlib._tight_layout import *  # noqa: F401, F403
 from matplotlib import _api
 _api.warn_deprecated("3.6", name=__name__, obj_type="module")
+
+
+@_api.deprecated("3.6", alternative="figure.canvas.get_renderer()")
+def get_renderer(fig):
+    if fig._cachedRenderer:
```

Comment:
This doesn't exist, so this function crashes immediately.

Reply:
OK! I have changed it. But is it currently enough to just do `fig.canvas.get_renderer()`?

Reply:
Btw, it seems like the original reporter managed to just copy this function (or he modified it trivially to deal with it).